docs/source/config.json

```diff
@@ -23,7 +23,12 @@
       "Security": {
         "CORS": "/configuration/cors",
         "CSRF prevention": "/configuration/csrf",
-        "JWT Authentication (experimental)": "/configuration/authn-jwt",
+        "JWT Authentication": [
+          "/configuration/authn-jwt",
+          [
+            "experimental"
+          ]
+        ],
```

Comment:
This broke our CI which didn't consider this expectation.  I've put it back how it was for the moment, and opened https://github.com/apollographql/router/issues/2651 to track that.

```suggestion
        "JWT Authentication (experimental)": "/configuration/authn-jwt",
```